modules/videoio/include/opencv2/videoio.hpp

```diff
@@ -650,6 +650,12 @@ class CV_EXPORTS_W VideoCapture
     */
     CV_WRAP virtual bool open(int index);
 
+    /**@overload
+
+    Parameters are open(int cameraNum,int apiPreference)
```

Comment:
Looks good to me except missing space after comma (3 locations):
> int cameraNum,<**space**>int apiPreference

And we will merge it.